pkgs/development/python-modules/swspotify/default.nix

```diff
@@ -11,7 +11,11 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = with python3.pkgs; [
     requests flask-cors dbus-python
-  ];
+  ] ++ lib.optionals stdenv.isDarwin [ python27Packages.pyobjc ];
+  
+  preConfigure = ''
+    substituteInPlace setup.py --replace 'requests>=2.24.0' 'requests>=2.23.0'
+  '';
```

Comment:
if you want to follow semver, which requests does
```suggestion
  postPatch = ''
    substituteInPlace setup.py \
      --replace 'requests>=2.24.0' 'requests~=2.23'
  '';
```

Reply:
Ok, done.